src/Illuminate/Database/Eloquent/Relations/MorphTo.php

```diff
@@ -253,6 +263,22 @@ public function getDictionary()
         return $this->dictionary;
     }
 
+    /**
+     * Specify which relations to load for a given morph type.
+     *
+     * @param array $with
```

Comment:
Use two spaces before and after the type:

```suggestion
     * @param  array  $with
```